benchmark/scripts/generate_harness/generate_harness.py

```diff
@@ -12,43 +12,70 @@
 #
 # ===---------------------------------------------------------------------===//
 
-# Generate CMakeLists.txt and utils/main.swift from templates.
+# Generate boilerplate, CMakeLists.txt and utils/main.swift from templates.
 
 from __future__ import print_function
 
-import glob
+import argparse
+import jinja2
```

Comment:
@palimondo I think the right thing to do here is to ignore jinja2 for now. Jinja2 is something that is already in tree. That being said, Dave has a good point here and we should consider ripping out Jinja2 in the future.

That is not IMO for this PR though. @dabrahams what do you think?